WcaOnRails/app/views/static_pages/_software_list.html.erb

```diff
@@ -0,0 +1,47 @@
+<% tools ||= {} %>
+<% clearfix = false %>
+<div class="row tools-list">
+  <% tools.each do |key, hash| %>
+    <div class="col-xs-12 col-md-6">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <h3 class="panel-title">
+            <%= hash[:title] %> <%= wca_icon if hash[:wst] %>
+          </h3>
+        </div>
+        <div class="panel-body">
+          <p><%= hash[:description] %></p>
+          <% if hash[:url] %>
+            <p>
+              <%= icon("fas", "link") %>
+              <%= link_to("Website", hash[:url], target: "_Blank") %>
```

Comment:
Nit: works either way, but it's the first time I see uppercase `_Blank`. Just pointing out ^^

Reply:
Ah no real reason actually, I'm not sure why I wrote it this :p